packages/flutter/test/rendering/editable_test.dart

```diff
@@ -406,20 +407,21 @@ void main() {
       offset: ViewportOffset.zero(),
       textSelectionDelegate: delegate,
       text: const TextSpan(
-        text: '中文测试文本是否正确',
-        style: TextStyle(
-          height: 1.0, fontSize: 10.0, fontFamily: 'Ahem',
-        ),
+        text: 'short',
```

Comment:
Does changing the text to english stop testing what we actually want to test? The bug specifically calls out Chinese characters.

Reply:
As far as I can tell the height of the caret doesn't depend on the script or the language system, and SkParagraph uses font metrics (ascent + descent) to compute the height. It wasn't working for Chinese characters before because we used [`preferredLineHeight` ](https://github.com/flutter/flutter/blob/8765492bd20867237bf5e3652eb746c21fe673df/packages/flutter/lib/src/painting/text_painter.dart#L768-L780) to compute the caret height and the default font used probably has the `0x20` space glyph but doesn't have Chinese glyphs so these Chinese characters are rendered using a fallback font that has a larger `ascent + descent` value.